connectors/agentic-ai/src/main/java/io/camunda/connector/agenticai/adhoctoolsschema/AdHocToolsSchemaExecutor.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. Licensed under a proprietary license.
+ * See the License.txt file for more information. You may not use this file
+ * except in compliance with the proprietary license.
+ */
+package io.camunda.connector.agenticai.adhoctoolsschema;
+
+import io.camunda.connector.agenticai.adhoctoolsschema.model.AdHocToolsSchemaResponse;
+import io.camunda.connector.agenticai.adhoctoolsschema.processdefinition.ProcessDefinitionAdHocToolElementsResolver;
+import io.camunda.connector.agenticai.adhoctoolsschema.schema.AdHocToolsSchemaResolver;
+
+public class AdHocToolsSchemaExecutor {
```

Comment:
:wrench: when I see the `Executor` suffix I expect an `execute()` method. Would it be possible to rename it?

Reply:
Yes, good point. Actually this was kind of redundant now (I had initially planned to use the executor from the Agent initializer as well), so I directly inlined it into the AdHocToolsSchemaFunction now.